backend/src/main/java/storage/PostgresSQLConnector.java

```diff
@@ -1,19 +1,22 @@
 package storage;
 
 
-
 import java.sql.*;
 import java.util.Properties;
 
 public class PostgresSQLConnector implements DBInterface {
 
-    private static final String CONNECTION_URL = "jdbc:postgresql://db:5432/svereadb";
+    private static final String CONNECTION_URL_DOCKER = "jdbc:postgresql://db:5432/svereadb";
+    private static final String CONNECTION_URL_DEBUG = "jdbc:postgresql://localhost:5432/svereadb";
     private static final String USERNAME = "admin";
     private static final String PWD = "1234";
+
+    private static String CONNECTION_URL;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "CONNECTION_URL" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)